tests/integration/multipass/test_multipass_provider.py

```diff
@@ -44,26 +44,36 @@ def test_create_environment(installed_multipass, instance_name):
     assert test_instance.exists() is False
 
 
-@pytest.mark.parametrize("alias", set(BuilddBaseAlias) - {BuilddBaseAlias.XENIAL})
+ALIASES = list(BuilddBaseAlias)
+ALIASES.remove(BuilddBaseAlias.XENIAL)
```

Comment:
In case anyone's curious why I made this change: doing this allows the use of `pytest-xdist` for these tests, since it gives us a guaranteed order.

Reply:
Nice!